politeiad/api/v1/v1.go

```diff
@@ -363,6 +378,22 @@ type InventoryReply struct {
 	Branches []Record `json:"branches"` // Last N branches (censored, new etc)
 }
 
+// InventoryByStatus requests for the censhorship tokens from all records
+// filtered by their status.
+type InventoryByStatus struct {
+	Challenge string `json:"challenge"` // Random challenge
+}
+
+// InventoryByStatusReply returns all censorship record tokens by status.
+type InventoryByStatusReply struct {
+	Response          string   `json:"response"`          // Challenge response
+	Unvetted          []string `json:"unvetted"`          // Unvetted censorship tokens
```

Comment:
Would prefer shorter comments.  The struct comment already says that they're all censorship record tokens so shortening the comment would make things cleaner while still giving all the same info.

![Screenshot from 2020-09-15 16-04-47](https://user-images.githubusercontent.com/9039877/93266172-4e946500-f76f-11ea-8105-3f46741414d4.png)

vs

![Screenshot from 2020-09-15 16-06-32](https://user-images.githubusercontent.com/9039877/93265139-c6fa2680-f76d-11ea-8993-7e691bd74d62.png)